docs/topics/cache.txt

```diff
@@ -875,6 +875,13 @@ of key-value pairs::
 
 Like ``cache.set()``, ``set_many()`` takes an optional ``timeout`` parameter.
 
+.. versionchanged:: 1.11
```

Comment:
Try to make the annotating "self-contained" as described in https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#documenting-new-features.